src/components/views/settings/tabs/user/LabsUserSettingsTab.tsx

```diff
@@ -63,9 +64,13 @@ export default class LabsUserSettingsTab extends React.Component<{}, IState> {
             this.setState({ showJumpToDate });
         });
 
+        MatrixClientPeg.get().doesServerSupportUnstableFeature("org.matrix.msc3827").
+            then((showExploringPublicSpaces) => this.setState({ showExploringPublicSpaces }));
```

Comment:
I think our code style says that the `.` and the `then` should be kept together on the second line